packages/web-components/fast-foundation/src/utilities/constructable.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @internal
+ */
+export type Constructable<T = {}> = {
```

Comment:
I think we could move this to fast-element and make it a public export. We've got multiple uses for this and I think it's something that fits in at the lowest level.